tests/__helpers__/test-utils.ts

```diff
@@ -2,7 +2,7 @@ import * as fs from 'fs';
 import { normalize } from 'jest-config';
 import * as setFromArgv from 'jest-config/build/set_from_argv';
 import * as path from 'path';
-import { JestConfigNormalize } from '../../src/jest-types';
+import { JestConfigNormalize } from '../../dist/types';
```

Comment:
this won't get the changed types right away, right?

Reply:
since they are test helpers, I must admit that it'd be more logic to take them from `src`, tho a build must be ran before we can start the tests anyway

Reply:
scratch that - the project gets built before tests so this should be fine